content/_policy/index._en.md

```diff
@@ -28,18 +28,14 @@ The information that you submit is used to create or update your Login.gov accou
+* Identity proofing requires more sensitive information such as a social security number, address, phone number, and U.S.-based state ID or driver’s license. A self-photograph is needed in cases where partners require verification through biometric comparison as further proof of your identity. Biometric comparison involves comparing your self-photograph to the portrait on your government-issued ID card.
+* When completing identity proofing in person at the United State Postal Service (USPS) Office, we share your validated name and address with USPS so that clerks can confirm your identity in person. 
 * When you visit Login.gov, we anonymously aggregate some data, which means that no individually identifying information is attached to it. We aggregate the pages visited, web browser type and version, and the length of the session to help us improve the Login.gov experience.For example, if you experience an error and we know your web browser type and version, we can better identify how to fix your issue.
 * Your stored account information is encrypted at two levels. Encryption translates your data into code that is only accessible by you, the Login.gov account holder. This means you are the only one who can view and make changes to your account information.
-* The Login.gov operators (i.e. our contact help center) cannot view your information or make any changes.
-* Information is maintained in accordance with GSA’s Records Retention Schedule, GRS 03.2.
-* **IRS Services Only.** When you access IRS services, Login.gov sends information about both successful and unsuccessful attempts during authentication and identity proofing, as well as any activities deemed to impact the security of your account. In contrast, Login.gov only sends successful attempts to other partner agencies. Each of these events is shared in a manner that only the IRS can access. Login.gov retains these events for no more than 168 hours and deletes event information once shared with the IRS. For further information as to how this attempts information is used by the IRS, please refer to the IRS’ Cybersecurity Data Warehouse Privacy and Civil Liberties Impact Assessment, [CSDW PCLIA](https://www.irs.gov/pub/irs-pia/csdw-pia.pdf).
+* The Login.gov operators (that is, our contact help center) cannot view your information or make any changes.
 
-Protecting your information is our priority. You can read more detailed explanations about [how Login.gov works](/policy/how-does-it-work/) and [our privacy act statement](/policy/our-privacy-act-statement/).
+Protecting your information is our priority. You can read more detailed explanations about [how Login.gov works](/policy/how-does-it-work/), [our privacy act statement](/policy/our-privacy-act-statement/), and [our security practices](/policy/our-security-practices/).
```

Comment:
The security practices page was removed in #1226 and I wouldn't expect to see these links being reintroduced here.